db/migrate/20170106153041_create_petition_detail_versions.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePetitionDetailVersions < ActiveRecord::Migration
+  def change
+    rename_table :petition_plugin_information, :petition_plugin_details
```

Comment:
Could you please move these to different migrations? Renaming the table, creating a new table and dropping columns.